The PR builder is failing due to something unrelated with these changes:

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (default-test) on project test: ExecutionException The forked VM terminated without properly saying goodbye. VM crash or System.exit called?
[ERROR] Command was cmd.exe /X /C "C:\Packages\Plugins\Microsoft.Compute.CustomScriptExtension\1.8\Downloads\0\tools\hudson.model.JDK\jdk8\jre\bin\java -Dfile.encoding=UTF-8 -Xmx1g -jar C:\Packages\Plugins\Microsoft.Compute.CustomScriptExtension\1.8\Downloads\0\workspace\Core_jenkins_PR-2851-7NNP5F22RZD2R5X4EAH2CSUY2YJI56S3FC4YDZ2LVB3IL262SFSA\test\target\surefire\surefirebooter156011780002022759.jar C:\Packages\Plugins\Microsoft.Compute.CustomScriptExtension\1.8\Downloads\0\workspace\Core_jenkins_PR-2851-7NNP5F22RZD2R5X4EAH2CSUY2YJI56S3FC4YDZ2LVB3IL262SFSA\test\target\surefire\surefire3610675378673748306tmp C:\Packages\Plugins\Microsoft.Compute.CustomScriptExtension\1.8\Downloads\0\workspace\Core_jenkins_PR-2851-7NNP5F22RZD2R5X4EAH2CSUY2YJI56S3FC4YDZ2LVB3IL262SFSA\test\target\surefire\surefire_58895678356336142184tmp"
[ERROR] -> [Help 1]
```

The tests were:

```
Tests run: 7877, Failures: 0, Errors: 0, Skipped: 5, Flakes: 1
```

--- a/core/src/main/java/hudson/diagnosis/OldDataMonitor.java
+++ b/core/src/main/java/hudson/diagnosis/OldDataMonitor.java
@@ -272,9 +272,9 @@ public class OldDataMonitor extends AdministrativeMonitor {
          * @return True if the major version# differs or the minor# differs by ≥ threshold
          */
         public boolean isOld(int threshold) {
-            return currentVersion != null && min != null && (currentVersion.digit(0) > min.digit(0)
-                    || (currentVersion.digit(0) == min.digit(0)
-                    && currentVersion.digit(1) - min.digit(1) >= threshold));
+            return currentVersion != null && min != null && (currentVersion.getDigitAt(0) > min.getDigitAt(0)
+                    || (currentVersion.getDigitAt(0) == min.getDigitAt(0)
+                    && currentVersion.getDigitAt(1) - min.getDigitAt(1) >= threshold));
         }
 
     }

